api.swagger.yaml

```diff
@@ -14511,6 +15200,171 @@ components:
           type: boolean
           default: false
           example: false
+    query-schema:
+      allOf:
+        - $ref: '#/components/schemas/query-schema-common'
+        - type: object
+          properties:
+            uuid:
+              description: 'Invotra UUID of the query.'
+              type: string
+              format: uuid
+              example: '01234567-89ab-cdef-1234-56789abcdef0'
+            attachment:
```

Comment:
queries do not have attachments. Could we double check all of the fields in the schemas in this PR exist on the CT? And in solr, as we could exclude them from the task if not, we only need the basic endpoint for now

Reply:
removed